app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    person_list = []
+    for person in people:
+        person_list.append(Person(person.get("name"), person.get("age")))
+    for i in range(len(people)):
+        if people[i].get("wife") is not None:
+            person_list[i].wife = Person.people[people[i].get("wife")]
```

Comment:
Do like this:
Person.people[person["name"]] = Person.people[person["wife"]]
Objects works by reference

Reply:
The same with husband

Reply:
Can you please check again?